sdk/cognitiveservices/Azure.InkRecognizer.WPF.Stroke/samples/NoteTakerWPF.csproj

```diff
@@ -0,0 +1,114 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For similar reasons as the UWP we should name this directory `sdk/cognitiveservices/InkRecongizier.WPF.Stroke`

Reply:
Done. Thanks